app/controllers/posts_controller.rb

```diff
@@ -17,9 +17,13 @@ def create
     user = current_user
```

Comment:
- [ ] Following the project requirements, you should solve N+1 problem when fetching all posts and their comments for a user as shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLAZ4K2ZHA777536NECDBHORQS).
For example in the posts#index actions, you should solve N+1 problem for the following query: `@posts = @user.posts`.

Reply:
Thank you @medaminedev66, I have fixed it.

Reply:
This issue is well resolved! ✔️